cromwell-drs-localizer/src/main/scala/drs/localizer/downloaders/BulkAccessUrlDownloader.scala

```diff
@@ -75,7 +75,7 @@ case class BulkAccessUrlDownloader(resolvedUrls: List[ResolvedDrsUrl]) extends D
     Files.deleteIfExists(getmManifestPath)
 
   def generateGetmCommand(pathToMainfestJson: Path): String =
-    s"""getm --manifest ${pathToMainfestJson.toString}"""
+    s"""timeout 4h getm --manifest ${pathToMainfestJson.toString} -vv"""
```

Comment:
Apologies in advance for the requirement whiplash; after talking to @dheiman we need the timeout pretty darn high to be safe. We do want to stop unmonitored VMs from running the full 14 days though.
```suggestion
    s"""timeout 24h getm --manifest ${pathToMainfestJson.toString} -vv"""
```